pyflakes/test/test_api.py

```diff
@@ -635,11 +610,10 @@ def test_encodedFileUTF8(self):
         """
         If source file declares the correct encoding, no error is reported.
         """
-        SNOWMAN = unichr(0x2603)
-        source = ("""\
-# coding: utf-8
-x = "%s"
-""" % SNOWMAN).encode('utf-8')
+        SNOWMAN = chr(0x2603)
```

Comment:
we're not fighting with py2 / py3.2 any more, it would be more readable to put the actual character in this string